lib/plurimath/math/function/unary_function.rb

```diff
@@ -24,7 +24,12 @@ def value_to_asciimath
         end
 
         def to_mathml_without_math_tag
-          "<mi>#{class_name}</mi>"
+          <<~LATEX
+            <mrow>
+              <mi>#{class_name}</mi>
+              #{parameter_one&.to_mathml_without_math_tag}
+            </mrow>
+          LATEX
```

Comment:
Again I don't think manually typing XML is the correct method to go but we'll see the other parts.